### PR TITLE
Fix Uglify minification error with Safari and ES6 

### PR DESCRIPTION
**Pull request by lcampanis**
Hi there,

Thought we'd let you know that because of an Uglify and Safari bug, Transliteration keeps breaking our app. The problem is with the following https://github.com/mishoo/UglifyJS2/issues/1753

However, as this module does use `const t = codemap[offset][ord];` https://github.com/andyhu/transliteration/blob/master/lib/node/transliterate.js#L77 It does force Uglify to break, and in turn Safari being strict with ES6, will break any app using Transliteration on any Safari mobile or desktop.

The following "suggestion" fixes this issue. 
In short better not use one char variables (except i and j for loops) and for Uglify to work as expected it would want those additional variables to sit outside.

Hope it helps.

credit @artlimes

### CODE REVIEW

**Comment by andyhu** [comment]
Hi, thanks a lot for your contribution! However, the file you have modified is the one that generated by gulp. The correct place should be `src/main/transliterate.js`. I've refactored the code mostly based on your commit in v1.6.0. Could you please check if it works for you? Thanks!

**Comment by andyhu** [comment]
Close for now since the problem should be solved in 1.6.x
